pythermalcomfort/models/work_capacity.py

```diff
@@ -0,0 +1,243 @@
+from typing import Union
+
+import numpy as np
+
+from pythermalcomfort.classes_input import (
+    WorkCapacityHothapsInputs,
+    WorkCapacityStandardsInputs,
+)
+from pythermalcomfort.classes_return import WorkCapacity
+
+
+def workcapacity_dunne(wbgt: Union[float, list[float]], intensity: str) -> WorkCapacity:
+    """
+    Estimate work capacity due to heat based ISO standards as described by Dunne et al
+
+    Estimates the amount of work that will be done at a given WBGT and
+    intensity of work as a percent. 100% means work is unaffected by heat. 0%
+    means no work is done.
+
+    This is based upon NIOSH safety standards. See:
+    Dunne JP, Stouffer RJ, John JG. Reductions in labour capacity from heat
+    stress under climate warming. Nature Climate Change. 2013 Jun;3(6):563–6.
+
+    Heavy intensity work is sometimes labelled as 400 W, but this is only
+    nominal. Medium work is assumed to be half as much as heavy intensity, and
```

Comment:
could you please provide the values med=200W. Could you please also explain what `nominal` means in this context?